geode-redis/src/test/java/org/apache/geode/redis/internal/SupportedCommandsJUnitTest.java

```diff
@@ -41,6 +41,10 @@
       "HMSET",
       "HSET",
       "HSETNX",
+      "HVALS",
```

Comment:
Looks like `HVALS` is in the unsupported and supported lists.

Reply:
this was the CI failure that confused me. it took a minute to figure out what was going on